### PR TITLE
Add tiles in Catalan

### PR DESCRIPTION
**Pull request by jmaspons**
See the map at https://www.openstreetmap.cat. Hosted by the [Breton OpenStreetMap Team](https://www.openstreetmap.bzh).

### CODE REVIEW

**Comment by brunob** [comment]
Nice (from bzh ^^), maybe we could put bzh & cat tiles in a common group in order to unify credits of the two layers ?

**Comment by jmaspons** [comment]
> Nice (from bzh ^^), maybe we could put bzh & cat tiles in a common group in order to unify credits of the two layers ?

That's fine for me, but I prefer that somebody else do that PR as I don't know much about the project and the format of the file.

**Comment by brunob** [comment]
Finally, i think attribution should simply be a copy of BZH tiles.

Question: does these tiles are restricted to specific bounds like bzh ? (can't find info about this for now)

**Comment by jmaspons** [comment]
> Question: does these tiles are restricted to specific bounds like bzh ? (can't find info about this for now)

Works worldwide (e.g. https://www.openstreetmap.cat/#map=10/43.0656/132.5715)

File: leaflet-providers.js [hunk 1]
```diff
@@ -121,6 +121,12 @@
 						bounds: [[46.2, -5.5], [50, 0.7]]
 					}
 				}
+				CAT: {
+					url: 'https://tile.openstreetmap.bzh/ca/{z}/{x}/{y}.png',
+					options: {
+						attribution: '{attribution.OpenStreetMap}, Tiles courtesy of <a href="https://wiki.openstreetmap.org/wiki/Ca:Mapa_en_catal%C3%A0" target="_blank">Breton OpenStreetMap Team</a>',
```
Comment (jmaspons) [inline leaflet-providers.js:127]:
```suggestion
						attribution: '{attribution.OpenStreetMap}, Tiles courtesy of <a href="http://www.openstreetmap.bzh/" target="_blank">Breton OpenStreetMap Team</a>',
```

Should I change the attribution link? I think that the current one is explicit enough and in the same language as the layer, so probably easier to read by the users.

**Comment by brunob** [comment]
> Works worldwide (e.g. https://www.openstreetmap.cat/#map=10/43.0656/132.5715)

Not sure about that => https://www.openstreetmap.cat/#map=5/37.1168/131.7194 doesn't show any label and zoom can't go beyond 5

File: leaflet-providers.js (hunk 1 above)
Comment (brunob) [reply leaflet-providers.js:127]: My bad, i think link should point to https://www.openstreetmap.cat, but i can't find any explicit attribution requirements for now :\

File: leaflet-providers.js [hunk 2]
```diff
@@ -121,6 +121,12 @@
 						bounds: [[46.2, -5.5], [50, 0.7]]
 					}
 				}
```
Comment (brunob) [inline leaflet-providers.js:123]: missing `,` here cf automated check fail ;)

**Comment by jmaspons** [comment]
> > Works worldwide (e.g. https://www.openstreetmap.cat/#map=10/43.0656/132.5715)
> 
> Not sure about that => https://www.openstreetmap.cat/#map=5/37.1168/131.7194 doesn't show any label and zoom can't go beyond 5

The lack of toponyms in zoom level 5 is a configuration of the renderer and don't depend on the area (happens the same at [Catalonia area](https://www.openstreetmap.cat/#map=5/44.4473/-0.8339))

File: leaflet-providers.js (hunk 2 above)
Comment (jmaspons) [reply leaflet-providers.js]: Thanks! Fixed in https://github.com/leaflet-extras/leaflet-providers/pull/562/commits/4b127491642883291f162c56e5a9389e115d59d7

File: leaflet-providers.js (hunk 1 above)
Comment (jmaspons) [reply leaflet-providers.js]: Changed in https://github.com/leaflet-extras/leaflet-providers/pull/562/commits/c2bb9884e8894f44868353e7506d6593fd10f725

**Comment by jmaspons** [comment]
Kindly ping, @brunob

**Review by brunob — APPROVED** [review]
Should be squashed on merge.

**Comment by brunob** [comment]
Ping @jieter

**Review by jieter — APPROVED** [review]

**Comment by brunob** [comment]
Thx for your patience @jmaspons :)

**Pull request merged**